### ISSUE TITLE
Error in getServerTiming -  undefined is not an object

### ISSUE THREAD

**Issue by petrvecera** [layer0-platform/rum#76]
Hi guys, I am getting a FE errors. 

They are coming from the RUM library on the Safari. 

```
TypeError
undefined is not an object (evaluating 'performance.getEntriesByType("navigation")[0].serverTiming')

const getServerTiming = () => {
    return performance.getEntriesByType('navigation')[0].serverTiming;
};
```

<img width="828" alt="image" src="https://github.com/layer0-platform/rum/assets/8086995/7b192aaf-390b-43a0-b321-df560460130c">

**Comment by petrvecera** [comment]
This is because `performance.getEntriesByType('navigation')` returns empty array on older Safari (14 and less)

**Comment by negyxo** [comment]
Hey Petr,

Thanks for letting us know. We'll fix this ASAP.

Best Regards man :)

**Comment by negyxo** [comment]
Petr,

This is released in v7.0.11.
